monai/bundle/utils.py

```diff
@@ -233,3 +242,22 @@ def load_bundle_config(bundle_path: str, *config_names: str, **load_kw_args: Any
             parser.read_config(f=cdata)
 
     return parser
+
+
+def merge_kv(args: dict | Any, k: str, v: Any) -> None:
+    """
+    Update the `args` dict-like object with the key/value pair `k` and `v`.
+    """
+    if k.startswith(MERGE_KEY):
+        id = k[1:]
+        if id in args and v is not None:
```

Comment:
Do you think we should add a warning here if id not in args but start with "+". Seems the output will be different based on the order of the config files.

Reply:
Well the output is supposed to be different depending on the order of config files as default is override ... 
Warning may be a good idea though.

Reply:
Added warning.